it/test_logs.py

```diff
@@ -66,6 +66,15 @@ def test_logs_streams(self, es_cluster, rally):
         )
         assert ret == 0
 
+    def test_logs_chicken(self, es_cluster, rally):
+        custom = {"mapping": "unmapped"}
+        ret = rally.race(
+            track="elastic/logs",
+            challenge="logging-chicken",
```

Comment:
This is `logging-insist-chicken`, see https://github.com/elastic/rally-tracks/blob/502a4112b7c0bd5a71c5fea5e52d2117ba027883/elastic/logs/challenges/logging-chicken.json#L3 We could rename `logging-chicken.json` to `logging-insist-chicken.json` to avoid this in the future.